qt/osm_auth_dialog.cpp

```diff
@@ -62,15 +61,14 @@ OsmAuthDialog::OsmAuthDialog(QWidget * parent)
   vLayout->addLayout(buttonsLayout);
 
   setLayout(vLayout);
-  setWindowTitle(isLoginDialog ? tr(kLoginDialogTitle) : tr(kLogoutDialogTitle));
+  setWindowTitle(tr(kLoginDialogTitle));
 }
```

Comment:
Why is this change necessary?

Reply:
Before, the header would be "OpenStreetMap Login Dialog" or "Logout Dialog" depending on if the user is logged in. 
The dialog doesn't actually change though, so it makes more sense to always show it as "OpenStreetMap Login"